### DISCUSSION TITLE
Hi, I think zotprime is a great thing and want to be a part of it

### DISCUSSION THREAD

**Discussion by eric-gitta-moore** [uniuuu/zotprime#11] [General]
Hi, I think zotprime is a really great thing and want to be a part of it too.

But before that, I want to know the situation first,

- Are there other deployment tools or docker images that are updated more frequently than this one?
- In addition to zotero, are there other relatively mature privatized deployment document management tools?

Thank you for your time

**Comment by uniuuu** [comment]
Hi @james-curtis 
- There could be more repositories which are similar or same . Please see in the README.md file at the below under `Credits` repositories found.
- This particular project focuses only on ZotPrime. 
Thank you.

**Comment by uniuuu** [comment]
Hi @eric-gitta-moore 
Good day. 
Try out latest code. 
More features have been added since last discussion and it's Zotero 8 client now.
